### ISSUE TITLE
Aqara Homekit Edition

### ISSUE THREAD

**Issue by monza258** [YinHangCode/homebridge-mi-gateway-security#14]
Hello,

Will it work with Homekit Edition ?

**Comment by rgerr** [comment]
Nope, you won’t be able to retrieve the so called token from the HomeKit gateway. Will work with the regular v3 gateway, though.
